### ISSUE TITLE
Bug: Services tab is empty (1.23.0)

### ISSUE THREAD

**Issue by theofficialgman** [hakandundar34coding/system-monitoring-center#100]
```
Traceback (most recent call last):
  File "/usr/share/system-monitoring-center/src/MainGUI.py", line 284, in on_main_gui_tab_radiobuttons_toggled
    self.main_gui_tab_switch_func()
  File "/usr/share/system-monitoring-center/src/MainGUI.py", line 518, in main_gui_tab_switch_func
    Services.services_initial_func()
  File "/usr/share/system-monitoring-center/src/Services.py", line 176, in services_initial_func
    services_loop_func()
  File "/usr/share/system-monitoring-center/src/Services.py", line 286, in services_loop_func
    service_load_state = systemctl_show_command_lines_split.split("LoadState=", 1)[1].split("\n", 1)[0].capitalize()
IndexError: list index out of range
```
nothing is listed in the services tab. starting from terminal this is output in terminal log ^

**Comment by theofficialgman** [comment]
this is actually related to https://github.com/hakandundar34coding/system-monitoring-center/issues/101

if the list is of length one or zero, you get the above error

**Comment by hakandundar34coding** [comment]
This bug is fixed and details can be found in #101.
